src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -3256,3 +3256,58 @@ def test_aks_update_label_msi(self, resource_group, resource_group_location):
             self.check('agentPoolProfiles[0].nodeLabels.label1', 'value11'),
             self.check('agentPoolProfiles[0].nodeLabels.label2', None),
         ])
+
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='centraluseuap')
+    def test_aks_create_with_oidc_issuer_enabled(self, resource_group, resource_group_location):
```

Comment:
What is the availability status of this feature? Is it only in canary or has it reached global? And does this feature support to be enabled through the custom header? If supported, please use custom header in the test case; otherwise, please add these two test cases to file `src/aks-preview/azcli_aks_live_test/configs/ext_matrix_default.json` to exclude them by default.

Reply:
It's in the global and it doesn't need a custom header.

Reply:
I had added these two tests to the matrix JSON. BTW, why can we just make use a decorator to mark a test as additional feature needed? I feel current process for adding a test is tend to miss steps.

Reply:
Good point and in fact, cli testsdk has such a `@record_only` decorator, but there are some problems and I have opened a PR trying to fix the issues.